curly.go

```diff
@@ -162,3 +168,46 @@ func (c CurlyRouter) computeWebserviceScore(requestTokens []string, tokens []str
 	}
 	return true, score
 }
+
+// Extract the parameters from the request url path
+func extractParameters(r *Route, urlParts []string) map[string]string {
```

Comment:
why did you move this definition to another file?

Reply:
To put it into the router as different routers treat params differently in their routing handling, as with the comment above I'll use this default behaviour if the router doesn't calculate the params itself